services/app-api/handlers/reports/update.ts

```diff
@@ -38,22 +44,27 @@ export const updateReport = handler(async (event, context) => {
     // if current report exists, get formTemplateId and fieldDataId
     if (getCurrentReport?.body) {
       const currentReport = JSON.parse(getCurrentReport.body);
-      const { formTemplateId, fieldDataId } = currentReport;
+      const { formTemplateId, fieldDataId, reportType } = currentReport;
       if (formTemplateId && fieldDataId) {
         // if report not in archived state, proceed with updates
         if (!currentReport.archived) {
-          const state: string = event.pathParameters.state;
+          const state: string = event.pathParameters?.state!;
```

Comment:
Nit: By the time we get here, `event.pathParameters.state` will always be defined. We used to be checking it above on line 24, but now that we're not, TS is mad, so it makes sense why we'd use `!` here to tell it so. But I think we don't need to use `?` for optional chaining here, because `event.pathParameters` will always be defined too. `event.pathParameters.state!` should work.